lib/src/HttpAppFrameworkImpl.cc

```diff
@@ -716,6 +716,19 @@ trantor::EventLoop *HttpAppFrameworkImpl::getLoop() const
     return &loop;
 }
 
+
+
+trantor::EventLoop *HttpAppFrameworkImpl::getIOLoop(const unsigned int &id) const
+{
+    assert(listenerManagerPtr_);
+    unsigned int id_( id );
+    if ( id_ >= threadNum_ ){
+        id_ %= threadNum_;
+LOG_WARN << "Invalid loop id : " << id << " will be rounded to : " << id_;
+    }
```

Comment:
These lines can be deleted because the same logic is implemented in the ListenerManager.

Reply:
Updated in the new commit